pyproject.toml

```diff
@@ -27,13 +27,14 @@ packages = ["torchqdynamics"]
 
 [tool.isort]
 profile = "black"
-skip = ["__init__.py"]
+skip = ["__init__.py", "venv"]
```

Comment:
I think we should rather specify the directories `isort` and `black` should explore (`tests` and `torchqdynamics`) rather than excluding the one they should avoid, let me fix it in a subsequent PR if you agree.

Reply:
See https://github.com/PierreGuilmin/torchqdynamics/pull/65.